src/short-css-class-names/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+* Returns an array with valid CSS class names that are between 1 and 3 chars long.
+*
+* Return value in pseudo code:
+* [
+*    "A",
+*    "B",
+*    ...
+*    "Z__",
+* ]
+**/
+module.exports = function() {
+    "use strict";
+
+    let charRange = require("./char-range"),
```

Comment:
The name `const` is horrible, but now it exists. It's my default choice.

Though, let's not get caught up in details. Go ahead and use let, and I'll use const :8ball: